subprojects/common/report-viewer/src/main/kotlin/com/avito/report/TestArtifactsProvider.kt

```diff
@@ -14,17 +14,17 @@ import java.io.File
  *
  * use [TestArtifactsProviderFactory] to create instances
  */
-interface TestArtifactsProvider {
+public interface TestArtifactsProvider {
```

Comment:
As I see this interface doesn't relate to `report-viwer`

Reply:
yes, seems it should be another module

Reply:
done